recipe/data.go

```diff
@@ -1738,9 +1738,9 @@ var AgeingSingleHouseholds = Response{
 					HRef:        "http://localhost:22400/code-lists/admin-geography",
 					IsHierarchy: true,
 				}, {
-					ID:          "ashe-sex",
+					ID:          "adult-sex",
```

Comment:
Is this going to break existing datasets?

Reply:
no it should not :)